### DISCUSSION TITLE
Details window able to expand to full screen

### DISCUSSION THREAD

**Discussion by Alister-Zeus** [hsoft/dupeguru#996] [Ideas]
I'm running a triple screen system and I suspect that dual screens are getting very common 

I'd like to have the details window full screen on one of the other screens I have (plus the option of setting the images scaled or 1:1 would be nice)

**Comment by glubsy** [comment]
Aren't you already able to do this? What version are you running?

**Comment by Alister-Zeus** [comment]
I’m on windows

 

You have to drag the window to the size you want, if you click on the title bar it snaps to the lower part of the main window

 

Running the latest version
